tailwind.config.js

```diff
@@ -7,12 +7,14 @@ module.exports = {
     extend: {
       animation: {
         "arrows-down": "arrows-down 2s infinite",
-        "slide-down": "slide-down 150ms ease-in-out",
-        "slide-up": "slide-up 150ms ease-in-out",
-        "pop-down": "pop-down 150ms ease-out forwards",
-        "pop-up": "pop-up 150ms ease-out forwards",
         "dialog-hide": "dialog-hide 150ms ease-out",
         "dialog-show": "dialog-show 150ms ease-out",
+        "pop-down": "pop-down 150ms ease-out forwards",
+        "pop-up": "pop-up 150ms ease-out forwards",
+        "slide-bg-right-slow": "slide-bg-right-slow 30s linear infinite",
+        "slide-bg-right-fast": "slide-bg-right-fast 15s linear infinite",
```

Comment:
These two can use the same keyframe:
```suggestion
        "slide-bg-right-slow": "slide-bg-right 30s linear infinite",
        "slide-bg-right-fast": "slide-bg-right 15s linear infinite",
```

Reply:
D'OH